source/grid/grid_generator.cc

```diff
@@ -7979,28 +7942,55 @@ namespace GridGenerator
             const auto bid = cell.face(f)->boundary_id();
             const auto mid = cell.face(f)->manifold_id();
 
+            // TODO - this isn't right. A line can have its own manifold id in
+            // 3D independent of whatever the face manifold ID is. That's where
+            // the bug comes from!
```

Comment:
It isn't clear to me what "this" in "this isn't right" refers to. Is it something above or below? Can you be more specific?

Reply:
that was a comment to myself that I meant to delete - the bug is the subject of the PR.

Reply:
OK. Let me know once you push an updated version!